docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,286 @@
+- Fix problems that are important to users
+- Engage the community in working toward the best possible code
+- Enable a sustainable system for SpECTRE's maintainers to review
+  contributions
+
+Please follow these steps to have your contribution considered by the
+maintainers:
+
+1. Follow all instructions in the template.
+2. Follow the \ref code_review_guide "code review guidelines", the
+  \ref writing_unit_tests "guide to writing unit tests", and the \ref
+  writing_good_dox "guide to writing documentation"
+3. After you submit your pull request, verify that all [status
+   checks](https://help.github.com/articles/about-status-checks/) are
+   passing
+
+   > If a status check is failing, and you believe that the failure is
+   > unrelated to your change, please leave a comment on the pull
+   > request explaining why you believe the failure is unrelated. A
+   > maintainer will re-run the status check for you. If we conclude
+   > that the failure was a false positive, then we will open an issue
+   > to track that problem with our status check suite.
+
+While the prerequisites above must be satisfied prior to having your
+pull request reviewed, the reviewers may ask you to complete
+additional design work, tests, or other changes before your pull
+request can be ultimately accepted.
+
+#### How SpECTRE pull request reviews are conducted
+
+> Note that these are guidelines and not rigid rules.
+
+In general, longer pull requests take longer to review, with the time
+scaling exponentially with the number of lines changed.  Therefore if
+your pull request is too large, we may ask you to break it up into
+several smaller pull requests.
+
+> Below, days mean business days, so if the time period includes the
+> weekend, add two days, and for major holidays add a day.
+> Furthermore, most of us are academics, and we occasionally go to
+> conferences which may lead to delays in the review process.  Also do
+> not expect much to happen between December 20th and January 3rd.
+
+Most pull requests submitted to SpECTRE will be reviewed in the
+following manner:
+- When creating the pull request, the author should add the
+  appropriate labels.  If the pull request is not a `new design` or
+  `in progress`, the author may either assign two reviewers (if GitHub
+  suggested any) or add the `review help wanted` label.
+- Within two days, assigned reviewers should either confirm that they
+  are able to review, or remove themselves while adding the `review
+  help wanted` label.
+- Anyone is welcome to self-assign themselves as a reviewer; when
+  there are at least two reviewers, the `review help wanted` label
+  should be removed.
+- If there are not two reviewers after two days, a SpECTRE core
+  developer will assign reviewers, removing the `review help wanted`
+  label.  If for some reason, no reviewers have been assigned after
+  three days, the author of the pull request should feel free to ping
+  `@sxs-collaboration/spectre-core-devs`
```

Comment:
It would be useful to somewhere explain whether the people in spectre-core-devs can change.  Specifically, is it possible/planned that dutifully spectre-contributors and reviewers will eventually be promoted to core-devs? I hope that's the case, and stating whatever expectation or thoughts might be around this, would make the structure more transparent and welcoming, and might encourage contributors to put in more effort.

Reply:
Yes, spectre-core-devs can change